protocol/data.md

```diff
@@ -112,16 +112,16 @@ inline void Encode(BinData &out) const {
 }
 ```
 
-The id: A universal identifier which is unique while the entity is alive, but once the entity is destroyed - the id may be used by another entity in the future. **IDs are unique, but can be recycled**
-The hash: Seems to be the total amount of times a specific id has been used/recycled.
+The id: A universal identifier that is unique while the entity is alive, but once the entity is destroyed - the id may be used by another entity in the future. **IDs are unique, but can be recycled**
+The hash: This seems to be the total amount of times a specific id has been used/recycled.
 
-**Quick Misconception,** in diep's console, sometimes you'll see messages relating to _"possible desyncs"_ and alongside them you'll see two numbers inside of these `<triangular bracket things>`, these are NOT `<hash, id>`. Unlike the order they are read, entids are logged as `<id, hash>` in the diep console.
+**Quick Misconception,** in Diep's console, sometimes you'll see messages relating to _"possible desyncs"_ and alongside them, you'll see two numbers inside of these `<triangular bracket things>`, these are NOT `<hash, id>`. Unlike the order they are read, entids are logged as `<id, hash>` in the Diep console.
 
 ---
 
 ### **Data Organization**
 
-Inside of a specifically 0x00 packets, there is a form of table which has come to be known as a `jumpTable`. Jump tables work differently than the standard arrays, by retrieving indexes from jumps, then parsing values based on index. The best way to explain the format is with an example, here is an example of a field group identification table, without any values.
+Inside of specifically 0x00 packets, there is a form of table which has come to be known as a "jump table. Jump tables work differently than the standard arrays, by retrieving indexes from jumps, then parsing values based on that index. The best way to explain the format is with an example, here is an example of a field group identification table, without any values.
```

Comment:
I am not sure if this paragraph here makes sense

Reply:
maybe rephrase to
```
Inside of 0x00 packets, there is data stored in a table-like form of data, which has come to be known as a "jump table". Jump tables work differently than standard arrays. They are parsed by retrieving indexes from jumps, then parsing values based on that index. The best way to explain the format is with an example, here is an example of a field group identification jump table, in this case, indexes without any values.
```

Reply:
Rephrased